test/cloudflare-handler.test.ts

```diff
@@ -109,10 +109,7 @@ describe("createCloudflareHandler(app)", () => {
   it("POST /api/github/oauth/token", async () => {
     const appMock = {
       createToken: jest.fn().mockResolvedValue({
-        authentication: {
-          token: "token123",
-          scopes: ["repo", "gist"],
-        },
+        authentication: { clientSecret: "" },
```

Comment:
why not leave the `token` and `scopes` properties in the test, only add `clientSecret`?

Reply:
Can you add `type: "token"` to `authentication` to test that we don't just set `authentication` to an empty object?

Reply:
Done.